plugin/web/extensions/storytelling/core/types.ts

```diff
@@ -0,0 +1,116 @@
+export type PostMessageType =
+  | "resize"
+  | "captureScene"
+  | "viewScene"
+  | "recaptureScene"
+  | "editScene"
+  | "closeSceneEditor"
+  | "saveScene"
+  | "getViewport"
+  | "shareStory"
+  | "saveStoryData"
+  | "cancelPlayStory";
```

Comment:
Can you tidy these up a bit and make sure it conforms to the other extensions' style.
ie.
- `type ActionType` (the PostMessageProps should be `{action: ActionType; payload?: any}:`
- try to name them consistently. For example, scene actions can be cleaned up to be `sceneXxxx`. That way we can have it like:
```
| "sceneView"
| "sceneCapture"
| "sceneRecapture"
| "sceneEdit"
..
..
```

Reply:
Updated.